src/interval.jl

```diff
@@ -351,6 +388,203 @@ true
 ≫(a, b) = greater_than_disjoint(a, b)
 # ≫̸(a, b) = !≫(a, b)
 
+###### Set-related Helpers #####
+
+# unbunch: represent one or more intervals as a sequence of endpoints and their indices into the original intervals
```

Comment:
Comment in the wrong place?

Reply:
Tried to improve the comments here to clarify